vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -355,6 +355,22 @@ void TextInputShadowNode::registerEvents() {
           }
         });
   }
+
+  control.as<xaml::UIElement>().AddHandler(
+      xaml::UIElement::PointerPressedEvent(),
+      winrt::box_value(xaml::Input::PointerEventHandler([=](auto &&, xaml::Input::PointerRoutedEventArgs const &args) {
+        folly::dynamic eventData = folly::dynamic::object("target", tag);
+        GetViewManager()->GetReactContext().DispatchEvent(tag, "topTextInputPressIn", std::move(eventData));
+      })),
+      true);
+
+  control.as<xaml::UIElement>().AddHandler(
+      xaml::UIElement::PointerReleasedEvent(),
+      winrt::box_value(xaml::Input::PointerEventHandler([=](auto &&, xaml::Input::PointerRoutedEventArgs const &args) {
+        folly::dynamic eventData = folly::dynamic::object("target", tag);
+        GetViewManager()->GetReactContext().DispatchEvent(tag, "topTextInputPressOut", std::move(eventData));
+      })),
+      true);
```

Comment:
normally you wouldn't use AddHandler directly, instead you'd do the same as above:
m_pointerPressedRevoker = element.PointerPressed( [] (auto&&, auto&& args) { ... } );

same w/PointerReleased

Is there a reason for using AddHandler instead of PointerPressed?

Reply:
Yes, the editable box within TextBox handles these events, and simply registering PointerPressed/Released does not work, as the event is already marked as handled; there is also no tunneling for this one, hence the use of AddHandler.